src/index.html

```diff
@@ -10,13 +10,111 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
```

Comment:
It seems there is a duplicate preconnect link to 'https://fonts.googleapis.com'. Consider removing the duplicate to clean up the code.